hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -136,6 +136,13 @@ public HoodieDefaultTimeline findInstantsAfter(String instantTime, int numCommit
         details);
   }
 
+  @Override
+  public HoodieDefaultTimeline findInstantsBefore(String instantTime) {
```

Comment:
Didn't realize you can have an overridden method with a return type which is a sub-type of original return type. Learnt something new today :)